sql/parse/parse.go

```diff
@@ -252,6 +252,8 @@ func convert(ctx *sql.Context, stmt sqlparser.Statement, query string) (sql.Node
 			convertAccountName(n.To...),
 			n.WithGrantOption,
 		), nil
+	case *sqlparser.Flush:
```

Comment:
Small nit but I'd put this after the `Revoke` statements. It's NBD at all, I just like seeing the `Revoke` statements come right after the `Grant` statements 🤣